src/main.rs

```diff
@@ -77,6 +78,24 @@ fn run() -> Result<(), failure::Error> {
                 .subcommand(
                     SubCommand::with_name("list")
                 )
+                .subcommand(
+                    SubCommand::with_name("delete-key")
```

Comment:
will revisit subcommands with stakeholders, these are probably not it.

Reply:
they are mos def not it